src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -104,6 +109,7 @@ public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody Upda
      * @return BulkCreateResult - Object representing the results of the Bulk Monument Validate operation
      */
     @PostMapping("/api/monument/bulk/validate")
+    @PreAuthorize(Authorization.isPartnerOrResearcher)
```

Comment:
Does using this also ensure that they are authenticated?

Reply:
Yes